templates/nsis/boringInstaller.nsh

```diff
@@ -7,13 +7,28 @@
 
 !ifndef BUILD_UNINSTALLER
   Function StartApp
-    ${StdUtils.ExecShellAsUser} $0 "$SMPROGRAMS\${PRODUCT_FILENAME}.lnk" "open" ""
+    ${GetParameters} $R0
+    ${GetOptions} $R0 "--update" $R1
+    ${IfNot} ${Errors}
+      ${StdUtils.ExecShellAsUser} $0 "$SMPROGRAMS\${PRODUCT_FILENAME}.lnk" "open" "--updated"
+    ${Else}
+      ${StdUtils.ExecShellAsUser} $0 "$SMPROGRAMS\${PRODUCT_FILENAME}.lnk" "open" ""
+    ${endif}
+  FunctionEnd
+
+  Function licensePre
```

Comment:
Could you please move `licensePre` function to [common.nsh](https://github.com/electron-userland/electron-builder/blob/master/templates/nsis/common.nsh) (after line `!ifndef BUILD_UNINSTALLER`)?